### PR TITLE
Fixes, settings file and test for project

### PR DESCRIPTION
**Pull request by c-goosen**
Fixes some issue with project around the new sandbox auth and adds a settings file for better setup and quick start.

Updated readme

Added todo list

Added a basic pytest

### CODE REVIEW

File: root.py [hunk 1]
```diff
@@ -1,17 +1,25 @@
 import os
 import requests
+from settings import settings
+from requests.auth import HTTPBasicAuth
+#import base64
```
Comment (boyan-soubachov) [inline root.py]: Extraneous comment?

File: settings/settings.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+from base64 import b64encode
+import base64
+SANDBOX = True
+
+SANDBOX_URL = "https://sandbox.root.co.za/v1/insurance"
+PROD_URL = "https://api.root.co.za/v1/insurance"
+    
+BASE_URL = SANDBOX_URL if SANDBOX else PROD_URL
+
+API_KEY = "sandbox_NTJkMGMzN2EtNmNiNS00ZTY1LWEyM2YtMGQ2YzkzNWY5ZmM3LjVzUFdBN1RZck42WHBQbG1SNlFQUS1PSkdjUENsSlFT"
```
Comment (boyan-soubachov) [inline settings/settings.py]: Is this your personal API key?

Comment (c-goosen) [reply settings/settings.py]: Yeah I forgot about it (it was late), but I removed it from my account already.

File: root.py (hunk 1 above)
Comment (c-goosen) [reply root.py]: Thanks will remove too.

**Comment by c-goosen** [comment]
@glasnoster Please review and reject or accept?

File: root.py [hunk 3]
```diff
@@ -1,17 +1,23 @@
...
+ROOT_APP_SECRET = settings.ROOT_APP_SECRET or os.environ.get('ROOT_APP_SECRET')
 
 
 class RootException(Exception):
     pass
 
 
 def list_gadget_models():
-    result = requests.get("%s/gadgets/models" % BASE_URL, auth=(ROOT_APP_ID, ROOT_APP_SECRET))
+    result = requests.get("%s/gadgets/models" % BASE_URL,
```
Comment (ncthbrt) [inline root.py]: @c-goosen Just a quick note, this method of authentication is marked for deprecation.
The new authentication method requires that either the `ROOT_API_KEY` be set to the username in basic auth, or that bearer token authentication be used (with the `Authentication: Bearer {{ROOT_API_KEY}}` header)

Comment (c-goosen) [reply root.py]: So its either or? Any of the two?

Comment (c-goosen) [reply root.py]: This makes more sense if I am honest: Authentication: Bearer {{ROOT_API_KEY}}

Comment (ncthbrt) [reply root.py]: Yip either one works

**Pull request merged**